packages/openapi-to-graphql/src/types/graphql.ts

```diff
@@ -46,9 +46,27 @@ export type ResolveFunction = (
   info: object
 ) => Promise<any> | any
 
+type SubscriptionContext = {
+  pubsub: any
+  [key: string]: any
+}
+
+export type SubscriptionIterator = (
+  root: object,
+  args: object,
+  ctx: SubscriptionContext,
+  info?: object
+) => AsyncIterable<string | string[]>
+
```

Comment:
Describing the subscribe method in a Subscription field, as described by graphQL.
There seems to be conflict on either using AsyncIterator or AsyncIterable [here](https://github.com/graphql/graphql-js/issues/1442) or [there](https://www.howtographql.com/graphql-js/7-subscriptions/) or [even here](https://github.com/davidyaha/graphql-redis-subscriptions/pull/114)